packages/ketcher-react/src/script/editor/tool/atom.ts

```diff
@@ -129,7 +139,11 @@ class AtomTool {
 
   mousemove(event) {
     const rnd = this.editor.render
+    const { layerX, layerY } = event
+
     if (!this.dragCtx || !this.dragCtx.item) {
+      // magic numbers to adjust the position of the icon relative to the mouse pointer
+      this.editor.hoverIcon.attr({ x: layerX - 9, y: layerY - 7 })
```

Comment:
Can't we measure it? :) 
May be functions like `getBBox()` or `getBoundingClientRect()`?

Reply:
or at least put these int constants?